### PR TITLE
Add to FLAdminAPI, fix FLAdminAPI runner

### PR DESCRIPTION
**Pull request by nvkevlu**
- Update FLAdminAPI Runner to work with provisioned startup package and fed_admin.json instead of individual args
- add abort_job command into FLAdminAPI
- add auth for delete_job and clone_job

### CODE REVIEW

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
Please make sure conflicts are resolved and integration tests are passing.
Thanks!

**Comment by nvkevlu** [comment]
Cancelled these tests after seeing them take over 2 hours and not complete. They did pass on my local, so not sure why these did not. It seems to be cuda memory issue?

**Comment by YuanTingHsieh** [comment]
Hi @nvkevlu ,

Thanks for the clarification.
Yes, the CI machine has CUDA OOM error because someone is running on it.
Got it, if it works on your machine then it is good.

File: nvflare/fuel/hci/client/api.py [hunk 1]
```diff
@@ -488,8 +488,10 @@ def do_command(self, command):
         elif len(entries) == 1:
             self.set_command_result(None)
             ent = entries[0]
-            ent.handler(args, self)
+            return_result = ent.handler(args, self)
```
Comment (YuanTingHsieh) [inline nvflare/fuel/hci/client/api.py:491]: when will this be used?

what client command has a return result?

Comment (YuanTingHsieh) [inline nvflare/fuel/hci/client/api.py:491]: when will this be used?

what client command has a return result?

Comment (nvkevlu) [reply nvflare/fuel/hci/client/api.py:491]: this is for the client side commands, so the File transfer, but now HA, the result is returned.

Comment (YuanTingHsieh) [reply nvflare/fuel/hci/client/api.py:491]: Got it, the "Admin client" side commands.

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Pull request merged**